product_docs/docs/eprs/7/08_xdb_cli/03_xdb_cli_commands/52_reload_conf_file.mdx

```diff
@@ -85,11 +85,17 @@ The table shows whether a configuration property can be reloaded.
 
 This example reloads the configuration file.
 
-```shell
-$ java -jar edb-repcli.jar -reloadconf -repsvrfile ~/subsvr.prop
+!!! Note Note
+   When you execute the reloadconf command, if any configuration options have been changed from their default values, the output includes the configuration option and its new value. 
 
+```shell
+java -jar edb-repcli.jar -reloadconf -repsvrfile subsvr.prop
+__OUTPUT__
 Reloading Subscription Server configuration file...
 Reloaded configuration options from ../etc/xdb_subserver.conf...
+The conf option 'snapshotParallelTableLoaderLimit' set to '1'
+The conf option 'skipCheckConst' set to 'false'
+The conf option 'snapshotParallelLoadCount' set to '1'
 
 Configuration was reloaded successfully.
 ```
```

Comment:
The following message is displayed at the end of the note in the current staging deployment version of this page:

> export const _frontmatter = {"title":"Reloading the publication or subscription server configuration file (reloadconf)"}

Reply:
fixed